vision6.module

```diff
@@ -485,14 +485,6 @@ function vision6_form_node_form_alter(&$form, &$form_state, $form_id) {
   // spec.
   drupal_add_js(drupal_get_path('module', 'vision6') . '/js/vision6.js', 'file');
 
-  if (user_access('access vision6 publish') && !user_access('publish revisions')) {
```

Comment:
Vision6 module should not rely on permissions of any other module. The 'publish revisions' is not Drupal core permission and not this module permission.